shell/ci/auth/github_packages.sh

```diff
@@ -55,5 +55,6 @@ if command -v npm >/dev/null 2>&1; then
   cat >>"$HOME/.npmrc" <<EOF
 
 //npm.pkg.github.com/:_authToken=$GITHUB_TOKEN
+@getoutreach:registry=https://npm.pkg.github.com
```

Comment:
suggestion: use existing variable
```suggestion
@$ORG:registry=https://npm.pkg.github.com
```